docs/manual/cluster-admin/basic-management-operations.md

```diff
@@ -151,4 +151,117 @@ Another example is to restart the whole cluster:
+
+### Configure A Self-Signed Certificate
+
+
+#### 1. Enter the dev-box onctainer
+
+You need to use [`paictl`](#pai-service-management-and-paictl) to enter the container in the following way:
+
+```bash
+sudo docker exec -it dev-box bash
+```
+
+#### 2. Create a folder in the dev-box container
+When you enter the container, you need to create a folder and generate a self-signed certificate in this folder. We can create an ssl folder in the home folder.
+
+``` bash
+mkdir /home/ssl
+cd  /home/ssl
+```
+#### 3. Generate RSA private key with openssl
+The next commands you enter will use the FileName parameter multiple times. You can choose a suitable file name to replace FileName.
+
+``` bash
+openssl genrsa -des3 -out FileName.key 1024
+```
+
+In this step, password will be asked.
+
+#### 4. Generate certificate request
+
+```bash
+SUBJECT="/C=US/ST=Washington/CN=FileName"
+openssl req -new -subj $SUBJECT -key FileName.key -out FileName.csr
+```
+
+#### 5. Generate certificate
+
+```bash
+mv FileName.key FileName.origin.key
+openssl rsa -in FileName.origin.key -out FileName.key
+openssl x509 -req -days 3650 -in FileName.csr -signkey FileName.key -out FileName.crt
+```
+
+#### 6. Final result
+
+In the current directory, you will find 4 files:
+
+<div  align="center">
+<img src="./imgs/aad/openssl_result.png" alt="paictl overview picture" style="float: center; margin-right: 10px;" />
+</div>
+
+#### 7. Configure services-configuration.yaml
+
+
+If you are configuring for the first time, `services-configuration.yaml` may not exist in the dev-box container. You should follow the procedure below to change the configuration file and make it effective. Close the pylon service, pull the OpenPAI configuration file `services-configuration.yaml` to the local, change the configuration file, upload the configuration file, and restart the pylon service. The commands you need are:
+```bash
+./paictl.py service stop -n pylon
+./paictl.py config pull -o <config-folder>
+vim <config-folder>/services-configuration.yaml
+./paictl.py config push -p <config-folder> -m service
+./paictl.py service start -n pylon
+```
+
+
+If you already have `services-configuration.yaml` in your container, you can omit the process of pulling the file. Please note that the master_ip in the configuration file is the IP of your master machine, not the IP of your dev box machine. Please configure the yaml file in the following format:
+```
+pylon:
+    port: 80
+    uri: "http://master_ip:80"
+    ssl:
+      crt_name: xxxxxx
+      crt_path: /path/to/xxxxxx
+      key_name: yyyyyy
+      key_path: /path/to/yyyyyy
+```
+In the example we just gave, the configuration file content should be:
+```
+pylon:
+    port: 80
+    uri: "http://master_ip:80"
+    ssl:
+      crt_name: FileName.crt
+      crt_path: /home/ssl/FileName.crt
+      key_name: FileName.key
+      key_path: /home/ssl/FileName.key
+```
+Restart the pylon service, you can access OpenPAI via https.
```

Comment:
`Restart the pylon service, you can access OpenPAI via https.` -> `Restart the pylon service, and you will be able to access OpenPAI via https.`